tests/factories/item_factories.py

```diff
@@ -15,6 +15,7 @@ class ItemFactory(SQLAlchemyModelFactory):
     summary = 'Summary of the feed item.'
     text = 'Text of the feed item.'
     feed = factory.SubFactory(FeedFactory)
+    totalVotes = 0
```

Comment:
That's what the default is for. If you want to build an item from the factory with a different number of votes, you'll be able to do ItemFactory(totalVotes=1000).